src/main/java/org/openlmis/requisition/web/BaseRequisitionController.java

```diff
@@ -243,9 +239,6 @@ InitiateResult doInitiate(UUID programId, UUID facilityId, UUID suggestedPeriod,
         program, facility, period, emergency, stockAdjustmentReasons,
         requisitionTemplate, approvedProducts);
 
-    profiler.start("VALIDATE_REASONS");
-    reasonsValidator.validate(stockAdjustmentReasons, newRequisition.getTemplate());
```

Comment:
we cannot remove validation just like that. I know that it makes sense for Malawi impl because of their requestion, however it should be more generic solution